src/ViewHoldingsRecord.js

```diff
@@ -667,6 +669,9 @@ class ViewHoldingsRecord extends React.Component {
       },
     ];
 
+    const isCentralTenant = checkIfUserInCentralTenant(stripes);
+    const suppressActionMenu = isLocalInstance && isCentralTenant;
```

Comment:
Hi @OleksandrHladchenko1 local instances can have only member tenants, is it required to suppress holdings' action menu from member tenants' local instances?